tests/system/python/conftest.py

```diff
@@ -668,6 +669,33 @@ def _disable_sch(fledge_url, sch_name):
     return _disable_sch
 
 
+@pytest.fixture(scope="function", autouse=True)
+def collect_support_bundle(request):
+    def _collect_support_bundle():
+        PROJECT_ROOT = subprocess.getoutput("git rev-parse --show-toplevel")
+        try:
+            conn = http.client.HTTPConnection("{}".format(url))
+            conn.request("POST", "/fledge/support")
+            r = conn.getresponse()
+            assert 200 == r.status
+            r = r.read().decode()
+            jdoc = json.loads(r)
+            assert jdoc["bundle created"]
+            subprocess.run(["mkdir -p {0}/support/ && cp -r {1} {0}/support/.".format(PROJECT_ROOT, jdoc["bundle created"])], shell=True, check=True)
+        except Exception as e:
+            print("\n Failed to get Support Bundle from Fledge REST API due to {}".format(str(e)))
+            print("Copying syslog")
+            OS_NAME = subprocess.getoutput('cat /etc/os-release  | grep -E ^NAME= | cut -d= -f2 | tr -d \\" ')
+            if OS_NAME == 'Ubuntu':
+                LOG_FILE = "/var/log/syslog"
+            elif OS_NAME == 'CentOS Stream':
+                LOG_FILE = "/var/log/messages"  
+            subprocess.run(["mkdir -p {0}/support/ && cp {1} {0}/support/syslog_{2}".format(PROJECT_ROOT, LOG_FILE, time.strftime("%Y_%m_%d_%H_%M_%S"))], shell=True, check=True)  
+         
+    url = request.config.getoption("--fledge-url")
```

Comment:
use fledge_url def for that

Reply:
I tried fledge_url def but that was not working, So I have used this `request.config.getoption("--fledge-url")`